lib/uber/api/places.rb

```diff
@@ -5,13 +5,15 @@
 module Uber
   module API
     module Places
-      def places(place_id)
+      def place(place_id)
```

Comment:
Modified this method name to keep it inferable. Here, we are retrieving details about a place.